test/e2e-test-application/src/luigi-config/extended/globalSearch.js

```diff
@@ -1,5 +1,6 @@
 class GlobalSearch {
   constructor() {
+    pwd;
```

Comment:
```suggestion
```
what was the intention here. It's not defined anywhere and it's making the Build's fail. I guess it's a typo